.mailmap

```diff
@@ -1,4 +1,4 @@
-Alexey Orlenko <eaglexrlnk@gmail.com>
+Alexey Orlenko <eaglexrlnk@gmail.com> Oleksii Orlenko <oleksii.orlenko@infotech.gov.ua>
```

Comment:
I think it's better to use public email for open-source projects

Reply:
/cc @aqrln

Reply:
Yeah, I forgot that my Git email was set to this one when I committed.